pkg/transforms/http.go

```diff
@@ -55,6 +55,9 @@ func (sender HTTPSender) HTTPPost(edgexcontext *appcontext.Context, params ...in
 		if errReadingBody != nil {
 			return false, errReadingBody
 		}
+
+		edgexcontext.LoggingClient.Trace(fmt.Sprintf("Data exported via HTTP. Corrilation Id is '%s'", edgexcontext.CorrelationID))
```

Comment:
Corrilation should be "Correlation"

Reply:
You could again use the correlation-id key/value here